### PR TITLE
MRG+1 bugfixes to plot_joint

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Just a bit of internal work: hopefully slightly cleaner code.

@jaeilepp maybe you can take a look later?..

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1238,10 +1240,8 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     else:
         locator = None
 
-    topomap_args_pass = dict((k, v) for k, v in topomap_args.items() if
-                             k not in ['times', 'axes', 'show', 'colorbar'])
-    topomap_args_pass['outlines'] = (topomap_args['outlines'] if 'outlines'
-                                     in topomap_args else 'skirt')
+    topomap_args_pass = topomap_args.copy()
+    topomap_args_pass['outlines'] = topomap_args.get('outlines', 'skirt')
```
Comment (jaeilepp) [inline mne/viz/evoked.py]: So this is deviating from the default args? I know it's the same as before, but maybe it should be documented.

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -1110,24 +1110,24 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
...
     topomap_args : None | dict
-        A dict of `kwargs` that are forwarded to `evoked.plot_topomap`
-        to style the topomaps. `axes` and `show` are ignored. If `times`
-        is not in this dict, automatic peak detection is used. Beyond that,
-        if ``None`, no customizable arguments will be passed.
-        Defaults to ``None``.
+        A dict of `kwargs` that are forwarded to `evoked.plot_topomap` to
+        style the topomaps. `axes`, `show`, `times`, `colorbar` are illegal.
+        If `None`, no customizable arguments will be passed.
+        Defaults to `None`.
```
Comment (jaeilepp) [inline mne/viz/evoked.py]: I'm not sure about the styling, but I probably wouldn't use backticks with None and use double backticks with the others.

File: mne/viz/evoked.py (hunk 1 above)
Comment (jona-sassenhagen) [inline mne/viz/evoked.py]: Yes, it's deviating from the defaults. I'll doc it.

**Comment by jona-sassenhagen** [comment]
I think the travis error is unrelated?

**Comment by jona-sassenhagen** [comment]
Wow, I can restart individual travis jobs now!

**Comment by jona-sassenhagen** [comment]
Reviews please? :)

**Comment by jaeilepp** [comment]
LGTM

**Comment by agramfort** [comment]
thx @jona-sassenhagen

**Comment by jona-sassenhagen** [comment]
Thanks!

**Pull request merged**